GeneratorInterface/ThePEGInterface/plugins/ThePEGHadronizer.cc

```diff
@@ -41,15 +49,22 @@ class ThePEGHadronizer : public ThePEGInterface, public gen::BaseHadronizer {
 	const char *classname() const { return "ThePEGHadronizer"; }
 
     private:
+
+        virtual void doSetRandomEngine(CLHEP::HepRandomEngine* v) override { setPEGRandomEngine(v); }
```

Comment:
It seems that the corresponding CLHEP::HepRandomEngine* v function is not available in CMSSW 5_3_X.